gui/build.py

```diff
@@ -118,14 +132,20 @@ def parse_args():
 
     bundle = subparsers.add_parser('bundle',
                                    help='bundle the application (freeze)')
-    bundle.add_argument('--python-path', dest='python_path',
-                        default='C:\\Python27',
-                        help='your python path on the'
-                        ' system(default: %(default)r)')
-    bundle.add_argument('--nsis-path', dest='nsis_path',
-                        default='C:\\NSIS',
-                        help='your NSIS path on the'
-                        ' system(default: %(default)r)')
+    if sys.platform == 'win32':
+        bundle.add_argument('--python-path', dest='python_path',
+                            default='C:\\Python27',
+                            help='your python path on the'
+                            ' system(default: %(default)r)')
+        bundle.add_argument('--nsis-path', dest='nsis_path',
+                            default='C:\\NSIS',
+                            help='your NSIS path on the'
+                            ' system(default: %(default)r)')
+    elif sys.platform == 'linux2':
+        bundle.add_argument('--python-path', dest='python_path',
```

Comment:
instead of copy/pasting the whole option definition, you should just define a default variable that is different on each os. Something like:
if IS_WIN:
   default_python_path = 'C:\\Python27'
else:
   default_python_path = '/usr/lib/python2.7'

bundle.add_argument('--python-path', default=default_python_path, ...)